src/ui/camera.js

```diff
@@ -156,15 +193,21 @@ class Camera extends Evented {
     }
 
     /**
-     * Pans the map to the specified location, with an animated transition.
+     * Pans the map to the specified location with an animated transition.
      *
      * @memberof Map#
      * @param lnglat The location to pan the map to.
-     * @param options Options object
+     * @param options Options describing the destination and animation of the transition.
```

Comment:
I made a not-really-conscious decision not to do optional parameters since they had been used so inconsistently in the past. At this point, adding them with any degree of consistency is such a big task that it's probably better to tackle them all at once as part of a future project. There are some other tasks, like ensuring we're using internal linking, that deserve a more holistic look as well. I can open an issue to track this kind of follow-up work and put it on the docs team's roadmap!

Reply:
Sounds good!